content/tutorials/publish-subscribe.textile

```diff
@@ -141,6 +199,38 @@ blang[ruby].
 
   "See this step in Github":https://github.com/ably/tutorials/commit/212b65b
 
+blang[php].
+  Create @publish.php@ file which will be responsible for publishing messages:
+
+  ```[php]
+    <?php
+    require_once 'ably.php';
+    if (isset($_POST['message']) && !empty($_POST['message'])) {
+        $channel = $ably->channels->get('sport');
+        $channel->publish('update', $_POST['message']);
+    }
+    ?>
+    <html>
+      <body>
+        <h1>Input message</h1>
+        <form method="post">
+          <input type="text" name="message">
+          <input type="submit" value="Publish!">
+        </form>
+      </body>
+    </html>
+  ```
+
+  And to simplify it even further, let's add @iframe@ so we can se result without the need to switch browser windows. Add code below just after the @<h1>Publish & Subscribe sample</h1>@:
```

Comment:
change to "... let's add an @<iframe>@ so that we can publish messages from the same browser window."